app/models/assignment.rb

```diff
@@ -708,21 +714,20 @@ def summary_csv(role)
       csv << headers[0]
       csv << headers[1]
 
-      groupings.each do |g|
+      students.each do |student|
+        g = student.accepted_groupings.first
```

Comment:
This is good but a bit unusual because we would normally expect many associated `accepted_groupings` per student. Add a comment saying that we've filtered to keep only the groupings for this assignment when defining `students` above